event_routing_backends/processors/xapi/constants.py

```diff
@@ -58,6 +59,19 @@
 XAPI_CONTEXT_COMPLETION_THRESHOLD = 'https://w3id.org/xapi/video/extensions/completion-threshold'
 XAPI_CONTEXT_SESSION_ID = 'https://w3id.org/xapi/openedx/extensions/session-id'
 
+XAPI_ACTIVITY_PRACTICE_EXAM = 'https://w3id.org/xapi/openedx/extension/practice-exam'
```

Comment:
For consistency with the top level activity can we make these `practice-assessment` / `proctored-assessment`? Is one needed for timed assessments as well?

Reply:
Yes, I think so

Reply:
Solved